### PR TITLE
added github-handle to Emily Eldar

### LINKED ISSUE #7191: Add github-handle for Emily Eldar in home-unite-us.md
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started Page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to create a single variable `github-handle` to hold the github handle for each member of the leadership team. Eventually `github-handle` will replace the `github` and `picture` variables, reducing redundancy in the project file.

### Action Items
- [x] Open the file `_projects/home-unite-us.md` in the IDE
- [x] Replace:
```
- name: Emily Eldar
```
with
```
- name: Emily Eldar
  github-handle:
```
- [x] Do not use a tab to indent `github-handle`. YAML doesn't allow tabs; it requires spaces.
- [x] Using docker, confirm that the appearance of the project webpage is unchanged at all screen sizes. The project webpage URL can be found below under Resources.

### For Merge Team
- [ ] Release the dependency for this issue in #5441. If all the dependencies have been completed, close that issue.

### Resources/Instructions
https://github.com/hackforla/website/wiki/project.md-file-template
https://jekyllrb.com/
For QA to validate change: https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md?plain=1
Project Webpage: https://www.hackforla.org/projects/home-unite-us.md

- This issue is part of #5441.

### PR DESCRIPTION
**Pull request by pluto-bell**
Fixes #7191

### What changes did you make?
  - Added 'github-handle' under Emily Eldar's name, found in the home-unite-us project

### Why did you make the changes (we will use this info to test)?
  - To reduce redundancy in the project file.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
No visual changes were made to the website.

### CODE REVIEW

**Comment by TheManTheMythTheGameDev** [comment]
Review ETA: 4 PM 8/8/24
Availability: Every day

File: _projects/home-unite-us.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ identification: "228981080"
 title: Home Unite Us
 description: The Home Unite Us project is developing a workflow management tool for community nonprofits to automate and scale their existing Host Home initiatives, prioritizing streamlining of caseworkers' repetitive workloads, reducing institutional bias, and effectively capturing crucial data. <br /><br />Host Home programs are centered around housing young people, 18 - 25 years old. Their approach focuses on low-cost, community-driven intervention by matching a willing host with a guest or group of guests, providing a stable housing environment for youths who are experiencing homelessness and seeking stable housing.
 image: /assets/images/projects/home-heart.png
-alt: 'Home Unite Us'
+alt: "Home Unite Us"
```
Comment (TheManTheMythTheGameDev) [inline _projects/home-unite-us.md]: I don't believe this change was requested in the issue.

**Review by TheManTheMythTheGameDev — CHANGES_REQUESTED** [review]
Good job with this pull request! The "to" and "from" branches are set up correctly, the relevant issue is linked correctly, and the necessary change has been successfully made.
A few changes I would like to request:
 - I see that you made additional edits to _projects/home-unite-us.md. What was the motivation behind these edits? Even if they are necessary, they should most likely be part of a different pull request.
 - If there are no visual changes made to the website, please do not screen shot code changes. Instead, delete the image section altogether and add a small explanation as to why there are no visual changes. You can say "- No visual changes to the website"
 - You don't need to specify "good first issue" as a reason for why you made the changes.

Thank you for your time and contributions!

**Review by t-will-gillis — COMMENTED** [review]
Hey @pluto-bell Great job on your first HfLA PR.
- Branches will be ok. For the next time, please include the issue number with the name of your branch, similarly to what is shown in `CONTRIBUTING.md` [Sect. 2.7.b](https://github.com/pluto-bell/website/blob/addHandle/CONTRIBUTING.md#27b-working-on-an-issue-2-create-a-new-branch-where-you-will-work-on-your-issue). In other words, your branch should be named something like: `add-github-handle-7191`. 
- The sections for what was changed and why are fine. You have provided brief, relevant answers here. You can remove the bullet for "- First good issue."
- For the "Screenshots" section, please remove the dropdown images after your comment "No visual changes were made to the website." 
- The code changes are good except as @TheManTheMythTheGameDev mentions, you don't want to go outside the scope of the issue. In the future, the best thing to do when you see things like this will be to bring it up with the issue writer and/or the merge team. Sometimes there is another issue for addressing what you see and sometimes there will be a good reason that the code is written in some way. (Confession- I have been guilty of making little edits like this also...)  

Thank you for working on this!

**Comment by pluto-bell** [comment]
@t-will-gillis  @TheManTheMythTheGameDev, just want to be clear on what happened with those extra changes. I didn't recall touching anything else so I did some digging and came to realize both line 6 and 114  got modified by some type of code formatter (akin to VS Code's "Prettier" extension).  I disabled my extensions so that shouldn't occur moving forward.

I returned the alt text to single quotes, instead of the modified double.
Happy to return the assets double quotes to single as well if that's desirable.

**Comment by t-will-gillis** [comment]
Hi @pluto-bell Thank you for making the edits. Understand about the auto-formatting. Yes, please change the url line 114 back to single quotes also.

**Comment by pluto-bell** [comment]
@t-will-gillis great, I've finalized the edits (: 

Should be good to go now!

**Review by t-will-gillis — APPROVED** [review]
Great, thanks for making the change @pluto-bell!

**Comment by pluto-bell** [comment]
> Good job with this pull request! The "to" and "from" branches are set up correctly, the relevant issue is linked correctly, and the necessary change has been successfully made. A few changes I would like to request:
> 
> * I see that you made additional edits to _projects/home-unite-us.md. What was the motivation behind these edits? Even if they are necessary, they should most likely be part of a different pull request.
> * If there are no visual changes made to the website, please do not screen shot code changes. Instead, delete the image section altogether and add a small explanation as to why there are no visual changes. You can say "- No visual changes to the website"
> * You don't need to specify "good first issue" as a reason for why you made the changes.
> 
> Thank you for your time and contributions!

Hi @TheManTheMythTheGameDev, I think this merge is currently waiting on your review? As it still says "Changes Requested" from you. I believe everything should be good to go but let me know if not.

**Comment by roslynwythe** [comment]
@perlaroyerc Please add a comment with your availability and ETA for this PR review

**Review by TheManTheMythTheGameDev — APPROVED** [review]
Hi @pluto-bell! Yes, everything looks good to me now. Sorry if I wasn't clear about which double quotes should be returned to single quotes, but it seems as if that has all been resolved. The description looks good too. Good work on this issue, approved!

**Pull request merged**